### PR TITLE
Resubmit with requested changes

### PR DESCRIPTION
**Pull request by NickBethune**

### CODE REVIEW

File: search.py [hunk 1]
```diff
@@ -120,31 +158,35 @@ def notes_on_beam(tree):
...
     # todo: implement ability to decide which attributes to check on a search-by-search basis
 
     if element1.tag == element2.tag:
         tag = element1.tag
 
         if tag == namespace + "note":
             # check pname and dur of note
-            if element1.attrib["pname"] == element2.attrib["pname"] and element1.attrib["dur"] == element2.attrib["dur"]:
+
+            if element1.attrib["pname"] == element2.attrib["pname"] :
```
Comment (justinnhli) [inline search.py]: Remove space before final colon

**Comment by NickBethune** [comment]
@justinnhli 
@crzeller11

File: tests.py [hunk 2]
```diff
@@ -53,14 +54,22 @@ def positive_test_get_creator():
     assert len(creator_list) != 0, "get_creator: creator (composer) not found"
 
 
+
+def positive_test_check_element_match():
+    """positive test for seeing if all elements in a file match themselves"""
+    _,root = prepare_tree('database/Chopin.xml')
```
Comment (justinnhli) [inline tests.py]: need space after comma

File: tests.py [hunk 3]
```diff
@@ -53,14 +54,22 @@ def positive_test_get_creator():
...
+    all_equal = True
+    unequalelt = None
+    for element in root_to_list(root):
+        if not check_element_match(element, element) :
```
Comment (justinnhli) [inline tests.py]: No space after colon

**Review by justinnhli — COMMENTED** [review]
Just three small changes. Let me know when you've pushed again.

**Comment by NickBethune** [comment]
Made the changes requested
@justinnhli

**Comment by justinnhli** [comment]
Thanks.

**Pull request merged**